crates/testlang/outputs/npm/package/wasm/generated/interfaces/nomic-foundation-slang-cst.d.ts

```diff
@@ -681,6 +681,10 @@ export class NonterminalNode {
    */
   isTerminalNode(): false;
```

Comment:
I suggest adding a couple of TypeScript unit tests for these two APIs.

Reply:
and/or add it to the documentation

Reply:
+1 on the docs, but I wonder if we should do that along the mutator API instead? I'm not sure if any users will use this on its own.